logstash-core/lib/logstash/logging/slow_logger/top_items.rb

```diff
@@ -0,0 +1,69 @@
+# encoding: utf-8
+require "logstash/namespace"
+require "set"
+require 'thread'
+
+module LogStash; module Logging; module Util
+
+  class Operation
+
+    attr_reader :threshold, :time
+
```

Comment:
The tests seem to show `Operation.new(string, integer)`. Is the first argument meant to be a string or an integer?
If string, consider the name `:operation` instead.
If an integer, the name `:threshold` is OK.